### ISSUE TITLE
Add screencasts page

### ISSUE THREAD

**Issue by jj0hns0n** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#21]

**Comment by anitagraser** [comment]
Could you clarify what your vision is for this page? Would it be part of the documentation effort, with a focus on tutorials? Or would it be more geared towards marketing and present the available features?

**Comment by ghost** [comment]
@anitagraser I would say both, but when filed I was initially thinking of tutorials.

**Comment by anitagraser** [comment]
What would you think about linking to a curated Youtube play list? I don't think our regular documentation team has the resources to do videos as well but we could of course link to existing material.

**Comment by jj0hns0n** [comment]
Yeah, that sounds great. But actually just embedding some of them here
might not be a bad idea.

It would be great to have actual screencasts in the future that either
highlight new features.

**Comment by anitagraser** [comment]
Seems like the material on Vimeo https://vimeo.com/search?q=qgis is better than what Youtube currently has to offer.

**Comment by mbernasocchi** [comment]
closing as stale,  If this is still relevant, please reopen
Cheers

**Comment by anitagraser** [comment]
@mbernasocchi how about putting a recording of your FOSS4G talk on new QGIS features on https://qgis.org/en/site/about/index.html and mark this resolved?

**Comment by mbernasocchi** [comment]
You mean https://media.ccc.de/v/bucharest-185-qgis-is-dead-long-live-qgis-the-very-best-new-features-of-qgis-3-x ?
no problem for me, but marketing-wise I don't think it is great, I guess the original idea was more to have a flashy video. 
I think we could do a sales dodle like  https://www.youtube.com/watch?v=Et8Vbh08aT8
